test/scales/timeScaleTests.ts

```diff
@@ -1,178 +1,227 @@
-
-      scale.autoDomain();
-      scale.domainMax(smallMax);
-      assert.throws(() => scale.domainMin(bigMin), Error);
+    describe("Auto Domaining", () => {
+      let scale: Plottable.Scales.Time;
+
+      beforeEach(() => {
+        scale = new Plottable.Scales.Time();
+      });
+
+      it("expands single value domains to [value - 1, value + 1] when auto domaining", () => {
+        let singleValue = new Date(2000, 5, 5);
+        let dayBefore = new Date(2000, 5, 4);
+        let dayAfter = new Date(2000, 5, 6);
+        scale.addIncludedValuesProvider(() => [singleValue]);
+        scale.autoDomain();
+        let domain = scale.domain();
+        assert.strictEqual(domain[0].getTime(), dayBefore.getTime(), "left side of domain was expaded by one day");
+        assert.strictEqual(domain[1].getTime(), dayAfter.getTime(), "right side of domain was expaded by one day");
+      });
+
+      it("can force the minimum of the domain with domainMin()", () => {
+        scale.padProportion(0);
+        let requestedDomain = [new Date("2015-05-01"), new Date("2015-07-01")];
+        scale.addIncludedValuesProvider(() => requestedDomain);
+
+        let minBelowBottom = new Date("2015-04-01");
+        assert.strictEqual(scale.domainMin(minBelowBottom), scale, "the scale is returned by the setter");
+        assert.strictEqual(scale.domainMin().getTime(), minBelowBottom.getTime(), "can get domainMin");
+        assert.strictEqual(scale.domain()[0].getTime(), minBelowBottom.getTime(), "lower end of domain was set by domainMin()");
+        assert.strictEqual(scale.domainMin().getTime(), minBelowBottom.getTime(), "returns the set minimum value");
+
+        let minInMiddle = new Date("2015-06-01");
+        scale.domainMin(minInMiddle);
+        assert.strictEqual(scale.domain()[0].getTime(), minInMiddle.getTime(),
+          "lower end was set even if requested value cuts off some data");
+
+        scale.autoDomain();
+        assert.deepEqual(scale.domain(), requestedDomain, "calling autoDomain() overrides domainMin()");
+        assert.strictEqual(scale.domainMin().getTime(), scale.domain()[0].getTime(),
+          "returns autoDomain()-ed min value after autoDomain()-ing");
+
+        let minEqualTop = scale.domain()[1];
+        let nextDay = new Date("2015-07-02");
+        scale.domainMin(minEqualTop);
+        let domain = scale.domain();
+        assert.strictEqual(domain[0].getTime(), minEqualTop.getTime(),
+          "lower end was set even if requested value is >= autoDomain()-ed max");
+        assert.strictEqual(domain[1].getTime(), nextDay.getTime(), "upper end is set one day later");
+
+        scale.domainMin(minInMiddle);
+        let requestedDomain2 = [new Date("2014-05-01"), new Date("2016-07-01")];
+        scale.addIncludedValuesProvider(() => requestedDomain2);
+        assert.strictEqual(scale.domain()[0].getTime(), minInMiddle.getTime(),
+          "adding another ExtentsProvider doesn't change domainMin()");
```

Comment:
It is indeed testing multiple scenarios, but they are all about "forcing the minimum of the domain with domainMin()". Do you think it is worth breaking it up into multiple tests? I cannot see anything that does not respect the test title